src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2222,6 +2222,15 @@ bool AdaptDispatch::SetCursorColor(const COLORREF cursorColor)
     return _pConApi->SetCursorColor(cursorColor);
 }
 
+// Method Description:
+// - Restore the cursor style to user's default setting.
+// Return Value:
+// True if handled successfully. False otherwise.
+bool AdaptDispatch::RestoreCursorStyleToUserDefault() noexcept
+{
+    return false;
+}
```

Comment:
Again I don't think we need a separate method for this, and ideally conhost would support some version of the "restore default" functionality too, but worst case the existing behaviour should remain unchanged. We definitely shouldn't be returning false without doing anything.